android/app/build.gradle

```diff
@@ -127,63 +152,64 @@ android {
         }
     }
     signingConfigs {
-        release {
```

Comment:
Shouldn't we keep the release configuration?

Reply:
yes, good catch!